themes/beaver/assets/css/components/c-infobox.css

```
@@ -0,0 +1,209 @@
+  -webkit-box-direction: normal;
+  -webkit-flex-direction: column;
+  -ms-flex-direction: column;
+  flex-direction: column;
+  flex-shrink: 1;
+  min-width: 1px;
+  max-width: 100%;
+  -webkit-box-flex: 1 1 auto;
+  -moz-box-flex: 1 1 auto;
+  -webkit-flex: 1 1 auto;
+  -ms-flex: 1 1 auto;
+  flex: 1 1 auto;
```

Biome (2.1.2)

[error] Unexpected shorthand property flex after flex-shrink



(lint/suspicious/noShorthandPropertyOverrides)